test/jdk/sun/jvmstat/testlibrary/utils.sh

```diff
@@ -178,7 +178,7 @@ kill_proc_common() {
         # check if it's still hanging around
         if [ $? -eq 0 ]
         then
-            # it's still lingering, now it it hard
+            # it's still lingering, now it is hard
```

Comment:
```suggestion
            # it's still lingering, now hit it hard
```

Reply:
@mernst, this suggestion appears relevant. Could you update the PR to address this?